src/main/java/org/apache/brooklyn/cloudfoundry/entity/service/mysql/PaasMySqlServiceDriver.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.cloudfoundry.entity.service.mysql;
+
+
+import org.apache.brooklyn.cloudfoundry.entity.service.VanillaPaasServiceDriver;
+
+import com.google.common.annotations.Beta;
+
+@Beta
+public interface PaasMySqlServiceDriver extends VanillaPaasServiceDriver {
+
+    public static final String JDBC_URL_PROPERTY = "jdbcUrl";
```

Comment:
Actually, it is a big question for me.
Reading about credentials composition in database services, https://docs.cloudfoundry.org/services/binding-credentials.html

`JDBC` entry is not specified, so I think I should compose the JDBC string from URI (whose pattern is specified as `DB-TYPE://USERNAME:PASSWORD@HOSTNAME:PORT/NAME`, and we only need to add `jdbc:` prefix.

WDYT?